src/api/user/auth.js

```diff
@@ -70,3 +70,38 @@ exports.confirm = wrapper(async (req, res, next) => {
   await user.update({ isVerified: true })
   res.json('Success confirmed')
 })
+
+// TODO need to destroy previous JWT Token ------------------
+exports.changePassword = wrapper(async (req, res, next) => {
+  const result = await jwtPromiseVer(req.body.token, secret)
```

Comment:
Change function name of `jwtPromiseVer` to something more meaningful like `verifyJwt`